platforms/web/src/index.tsx

```diff
@@ -3,6 +3,8 @@ import { createRoot } from 'react-dom/client';
 import 'wicg-inert';
 import { registerSW } from 'virtual:pwa-register';
 import { configureEnv } from '@jwp/ott-common/src/env';
+import 'core-js/es/array/flat-map';
+import 'core-js/es/object/from-entries.js';
```

Comment:
Not sure if this import gives a problem when building...

```suggestion
import 'core-js/es/object/from-entries';
```

Reply:
Sloppy copy/paste mistake. Fixed.